### ISSUE TITLE
-r option in chisel_cloning isn't recognized

### ISSUE THREAD

**Issue by marialitovchenko** [raphael-group/chisel#15]
Dear developers, 

I would like to use -r option in chisel_cloning command as described on the [help page](https://github.com/raphael-group/chisel/blob/master/man/chisel-cloning.md) However, then I run following command:

`(chisel) [chisel]$ chisel_cloning -s 5  -f 0.1 --seed 123 -r 1`

I get an "unrecognized arguments: -r" error:
```
[2020-Sep-21 14:54:38]Parsing and checking arguments
usage: chisel_cloning [-h] [-x RUNDIR] [-f MAXDIFF] [-s MINSIZE] [-K UPPERK]
                      [--seed SEED]
                      [INPUT]
chisel_cloning: error: unrecognized arguments: -r

```

Could you please return the missing argument back?

**Comment by simozacca** [comment]
This is indeed a bug as the mentioned argument is missing from the `chisel_cloning` command; we will fix this bug in the upcoming release of CHISEL.

Thank you for reporting this issue!

**Comment by simozacca** [comment]
This bug has been fixed in release 0.0.5
